activesupport/lib/active_support/number_helper/number_to_rounded_converter.rb

```diff
@@ -1,12 +1,11 @@
 module ActiveSupport
   module NumberHelper
-    class NumberToRoundedConverter < NumberConverter # :nodoc:
-      self.namespace      = :precision
-      self.validate_float = true
-
-      def convert
-        precision = options.delete :precision
+    class RoundingHelper # :nodoc:
+      attr_reader :number, :options
 
+      def initialize(number, options)
+        @number = number
+        @options = options
         if precision
           case number
```

Comment:
Good call on moving this part into a constructor. It also means that line 7 (`@number = number`) can be moved to an `else` clause to the `if precision` call.